### DISCUSSION TITLE
Does (py)MuPDF support PDF/VT standard (Variable data printing)?

### DISCUSSION THREAD

**Discussion by hanssonrickard** [pymupdf/PyMuPDF#3990] [Q&A]
Hi.
I am looking for a way t generate PDF/T files where we insert variable data for thousands of records to be printed.
All according to PDF/VT standard.
Dues MuPDF support that?
It seems to be hard to find info on what PDF versions are supported.

**Comment by JorjMcKie** [comment]
Please direct this question to the MuPDF experts on [this](https://discord.gg/AwtN7GFv) Discord channel.
